src/main.ts

```diff
@@ -1,8 +1,11 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { ValidationPipe } from '@nestjs/common';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  // need to add ValidationPipe() to validate entity
+  app.useGlobalPipes(new ValidationPipe({ whitelist: true }));
```

Comment:
what is the purpose of `{ whitelist: true }` ?

Reply:
after discussion, keep the original version